protocol.md

```diff
@@ -160,6 +165,7 @@ NoteCutObject = {
 		Number, // Z value
 	],
 	"cutDistanceToCenter": Number, // Distance from the center of the note to the cut plane
+	"cutDistanceInverted": Boolean, // Whether the cut plane is on the negative side of the center of the note
```

Comment:
If you look at it objectively, cutDistanceToCenter and cutDistanceInverted have distinct meaningful uses (as part of the score calculation and slash offset calculation), but notePosition and cutPoint both don't (because they are both in world space coordinates, which have no meaning outside the game). If you are willing to run a small risk of breaking compatibility but don't want to modify cutDistanceToCenter to allow negative values (since it's useful so it might be in use), then instead how about not including the new fields, removing cutPoint and replacing it with cutPointLocal (cutPoint - notePosition)? Then only useful fields will be included.